packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -166,23 +166,61 @@ export default {
                 const offchainVerification = await this.offchainVerifier.verify(verification.hash)
 
                 if (offchainVerification) {
-                    if (!verification.issuerAddress && offchainVerification.status === 'registering') {
+                    if (!verification.events && offchainVerification.status === 'registering') {
                         // File not found on blockchain and offchain status is registering
+                        const identityVerifier = {}
+                        if (offchainVerification.issuerVerifiedBy) {
+                            identityVerifier.name = offchainVerification.issuerVerifiedBy
+                        }
+                        if (offchainVerification.issuerVerifiedImg) {
+                            identityVerifier.image = offchainVerification.issuerVerifiedImg
+                        }
+
                         verification = {
                             ...verification,
-                            ...offchainVerification
+                            ...offchainVerification,
+                            events: [{
```

Comment:
Why do we need to artificially create an register event here?
Or is this just temporary because we don't yet have the events in the offchainVerification results for registering files?

Reply:
Yes, that's temporary, I will add a comment